.github/workflows/release.yaml

```diff
@@ -26,15 +26,49 @@ jobs:
           role-to-assume: ${{ secrets.AWS_SAR_PUBLISHER_ROLE }}
           aws-region: us-east-2
 
+      - name: Build and Package Go executable
+        run: |
+          cd src
+          GOOS=linux GOARCH=amd64 go build -o bootstrap main.go
+          zip -r ../new-relic-log-forwarder.zip .
+
+      - name: Upload lambda code to S3 buckets in all AWS region
+        env:
+          file_to_upload: new-relic-log-forwarder.zip
+        run: |
+          regions=("us-east-1" "us-east-2" "eu-west-1" "eu-west-2")
+          #"us-west-1" "us-west-2" "af-south-1" "ap-east-1" "ap-south-2" "ap-southeast-3" "ap-southeast-5" "ap-southeast-4" "ap-south-1" "ap-northeast-3" "ap-northeast-2" "ap-southeast-1" "ap-southeast-2" "ap-northeast-1" "ca-central-1" "ca-west-1" "eu-central-1" "eu-south-1" "eu-west-3" "eu-south-2" "eu-north-1" "eu-central-2" "me-south-1" "me-central-1" "il-central-1" "sa-east-1")
+          for region in "${regions[@]}"; do
+            bucket_name="unified-lambda-logging-${region}"
```

Comment:
nit: Does this bucket exist in logint account? We could have added test also in name, but as this is a temporary thing we can keep it as it is.